<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN-L2504" ignored="0" name="Cash Flow Summary" tests="19" failures="16" timestamp="18 May 2021 20:50:26 GMT" time="1332.715" errors="0">
  <testcase name="testGetNotificationsForCashFlowSummary" time="142.515" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary"/>
  <testcase name="testGetNotificationsForCashFlowSummary" time="91.968" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary"/>
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.551" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.508" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.526" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.526" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.672" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="62.104" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.835" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.637" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.587" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.511" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.513" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.568" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.552" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.979" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.55" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="61.588" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary">
    <failure type="java.lang.AssertionError" message="TEST FAILED">
      <![CDATA[java.lang.AssertionError: TEST FAILED
	at com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary.testGetNotificationsForCashFlowSummary(TestGetInsightsForCashFlowSummary.java:298)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testGetNotificationsForCashFlowSummary -->
  <testcase name="testGetNotificationsForCashFlowSummary" time="91.989" classname="com.yodlee.app.yodleeApi.Insights.Notifications.V2.TestGetInsightsForCashFlowSummary"/>
</testsuite> <!-- Cash Flow Summary -->
